modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/KBDropdownPropsTransformer.js

```diff
@@ -28,12 +42,87 @@ const ACTIONS = {
 		});
 	},
 
-	move({moveItemUrl}) {
-		openModal({
+	move(
+		{
+			moveItemActionUrl,
+			moveItemModalUrl,
+			selectedItemClassNameId,
+			selectedItemId,
+			selectedItemType,
+		},
+		portletNamespace
+	) {
+		openSelectionModal({
+			buttonAddLabel: Liferay.Language.get('move'),
 			height: '50vh',
+			multiple: true,
+			onSelect: ({index, item, parentItem}) => {
+				if (!item) {
```

Comment:
This should not be needed. Do not send the current `item` from the modal, you have it already here . 
Also, I would rename things here: 
Use `item` for the one that selected the move option. So you should receive as arguments `itemClassNameId`, _itemId_ and _itemType_. And reserve `selectedItem` for the destination, what you select in the modal (this is the pattern we usually follow in the Item Selector). In this case, you are destructuring it, so you could keep the `onSelect:({index, parentItem})`. Otherwise it would be `onSelect: (selectedItem)` and then you access to `selectedItem.index` and `selectedItem.parentItem ` (as I said, it is just a convention we have)